lib/kernel/doc/src/pg.xml

```diff
@@ -89,7 +89,9 @@
       If there is another process registered under this name, or another ETS table
       exists, scope fails to start.</p>
       <p>Local membership is not preserved if scope process exits and
-        restarts.
+        restarts.</p>
+      <p>A scope can be kept local-only by using a scope name that is unique
+        cluster-wide.
```

Comment:
It might be useful to give an example here:

"<p>A scope can be kept local-only by using a scope name that is unique
        cluster-wide, e.g. node name: `pg:start_link(node())`."

Reply:
Done, thanks!